.github/workflows/pre-release.yml

```
@@ -217,6 +229,9 @@ jobs:
       - id: docker_build_image
         name: "Docker Build keptn/${{ matrix.config.artifact }}"
         uses: docker/build-push-action@v2
+        env:
+          DATETIME: ${{ steps.get_datetime.outputs.DATETIME }}
+          GIT_SHA: ${{ steps.extract_branch.outputs.GIT_SHA }}
```

When relying on data from a previous job you need to reference the outputs in a different way:

```suggestion
          DATETIME: ${{ needs.prepare.outputs.datetime }}
          GIT_SHA: ${{ needs.prepare.outputs.git_sha }}
```

---

and then you also need to add the outputs to the `prepare` job like so:
```
prepare:
  outputs:
    datetime: ${{ steps.get_datetime.outputs.DATETIME }}
    git_sha: ${{ steps.extract_branch.outputs.GIT_SHA }}
```